pkg/pillar/types/zedroutertypes.go

```diff
@@ -128,6 +128,53 @@ func (status AppNetworkStatus) VerifyFilename(fileName string) bool {
 			fileName, expect)
 	}
 	return ret
+
+}
+
+// IntfStatusMap - Used to return per-interface errors.
```

Comment:
s/error/status/

Reply:
Cleaned up the comment in PR: https://github.com/lf-edge/eve/pull/890